testutil/keeper/fungible.go

```diff
@@ -43,8 +43,8 @@ func FungibleKeeper(t testing.TB) (*keeper.Keeper, sdk.Context) {
 
 	bankkeeper := bankkeeper2.BaseKeeper{}
 	authkeeper := authkeeper2.AccountKeeper{}
-	evmKeeper := &evmkeeper.Keeper{}
```

Comment:
Why this change?

Reply:
This seems to be a mistake. Will update it.

Reply:
@lumtis addressed this one. please review.